tests/Sprite.cpp

```diff
@@ -503,7 +503,7 @@ TEST_CASE("[Sprite]")
 
     SECTION("getTexture has a version to change the texture and a const version")
     {
-        sprite.getTexture().setSmooth(false);
+        //sprite.getTexture().setSmooth(false);
```

Comment:
Was this commented for a reason?

Reply:
TGUI_NEXT deleted procedure that could return non const value and build was impossible in deprecated code removal.

Reply:
Removing deprecated code isn't documented so its normal that you didn't know, but to remove deprecated code you should define `TGUI_REMOVE_DEPRECATED_CODE`. The `TGUI_NEXT` define had a different goal (and it is no longer relevant). Once I've merged your changes I'm going to remove the TGUI_NEXT stuff to avoid further confusion.

So this line should be uncommented.